ref/general/authentication.adoc

```diff
@@ -13,16 +13,20 @@
 
 For example, a user of a music streaming service might want to listen to a song, learn about the artist, and get recommendations for similar artists. Each of these tasks would require access to a different service, but having to authenticate for each of them separately uses unnecessary time and resources. Open Liberty supports a number of different strategies for single sign-on (SSO) authentication so that a user or service must authenticate only once to access the various resources in an application. For example, the link:/docs/ref/feature/#socialLogin-1.0.html[Social Media Login] feature enables users of an application that is running on Open Liberty to authenticate through their social media login. After authentication, an application can either obtain a JSON Web Token (JWT) from the social media service or associated OpenID Connect provider, or create a new one. This JWT can then be used to authenticate the user to any service in the application.
 
-image::/docs/img/Authentication.png[Authentication diagram,align="center"]
+image::/docs/img/Authentication-rvsd.png[Authentication diagram,align="center"]
 
 == User registries
-A user registry is a database that contains information about a set of users and their associated roles. When a user requests a protected resource in an application, the user's credentials are checked against the information in one or more user registries. Open Liberty provides an easy-to-use link:/docs/ref/general/#basic-registry.html[basic user registry] for developers to use in test environments. Typically, application servers are configured to use an external registry, such as a link:/docs/ref/general/#LDAP-registry.html[Lightweight Directory Access Protocol (LDAP) registry]. Open Liberty also supports federated registries so that a server can be configured to authenticate from multiple registries.
+A user registry is a database that contains information about a set of users and their associated roles. When a user requests a protected resource in an application, the user's credentials are checked against the information in one or more user registries. Open Liberty provides an easy-to-use link:/docs/ref/general/#basic-registry.html[basic user registry] for developers to use in test environments. Typically, applications in production are configured to use an external registry, such as a link:/docs/ref/general/#LDAP-registry.html[Lightweight Directory Access Protocol (LDAP) registry].
+
+Open Liberty also supports link:/docs/ref/general/#federated-registries.html[federated user registries] so that a server can be configured to authenticate from multiple registries from a single logical view. Information from LDAP, basic, and custom user registries can be federated into a common registry to manage application security.
```

Comment:
I would probably swap this around and say that "Open Liberty supports multiple user registries and federates them into a single logical view." I think starting with what is possible and then introducing the term we use is probably easier to follow.